### PR TITLE
Use a better solution to fix the ID test that doesnt work in local

### PR DESCRIPTION
**Pull request by DiegoVazquezNanini**
This PR fix an issue with some ID tests that fail if you run the tests locally.
@jfsoul @markjamesbutler @TBonnin

### CODE REVIEW

**Comment by TBonnin** [comment]
Something like this https://gist.github.com/TBonnin/7b11368f841796dbcb286c5fe4e12cec in addition to your changes of `IdApi` and IdApiClient` should do the job.
Then in all the tests involving identity config you can define your own implementation of the `IdConfig` trait

**Review by TBonnin — CHANGES_REQUESTED** [review]
Something like this https://gist.github.com/TBonnin/7b11368f841796dbcb286c5fe4e12cec, in additions to your changes of `IdApi` and `IdApiClient`, would be cleaner.

Then in all the tests involving identity config you can provide your own implementation of the `IdConfig` trait, the same way we are providing `TestWsClient` for instance.

**Comment by DiegoVazquezNanini** [comment]
@TBonnin please let me know what do you think now and please let me know if you have further suggestions.

File: identity/test/package.scala [hunk 1]
```diff
@@ -42,3 +44,18 @@ class IdentityTestSuite extends Suites(
...
+
+trait WithTestIdConfig {
+  class IdentityConfigurationStub extends IdConfig {
+    val apiClientToken = "frontend-dev-client-token"
+    val apiRoot: String = ???
+    val accountDeletionApiRoot: String = ???
+    val accountDeletionApiKey: String = ???
+    val url: String = ???
+    val oauthUrl: String = ???
+    val domain: String = ???
```
Comment (TBonnin) [inline identity/test/package.scala]: You are gonna need real values here

Comment (DiegoVazquezNanini) [inline identity/test/package.scala]: Yeap, I'm working on a fix now.

**Review by TBonnin — APPROVED** [review]

**Comment by DiegoVazquezNanini** [comment]
Thanks for the review @TBonnin

**Pull request merged**